plugins/shared-react/src/components/common/StatusIconAndText.css

```diff
@@ -22,9 +15,3 @@
 .bs-shared-dashboard-icon {
   font-size: 1.2rem;
 }
```

Comment:
Is it okay to leave other styles in the CSS files? If we leave them in where they are now, later we may have other UI issues, what do you think? We can address those in a different story for sure, because I think it could be a common issue for other plugins which depends on shared-react.

cc. @christoph-jerolimov

Reply:
removed the css file, I checked those classes aren't required anymore